app/models/like.rb

```diff
@@ -0,0 +1,11 @@
+class Like < ApplicationRecord
+    belongs_to :user
+    belongs_to :post
+
+    after_create :update_like_counter
+  
+    def update_like_counter
+      post.update(likes_counter: post.likes.count)
+    end
```

Comment:
- [ Optional ] Kindly I recommend you to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #`update_like_counter `method every time a new post has been saved to database, then make the `update_like_counter` method private.

 ```
after_save : update_like_counter

  private

  def update_like_counter
      post.update(likes_counter: post.likes.count)
    end
```